tests/component/sections/dataset/dataset-publish/PublishDatasetModal.spec.tsx

```diff
@@ -79,10 +81,12 @@ describe('PublishDatasetModal', () => {
           repository={repository}
           persistentId="testPersistentId"
           releasedVersionExists={true}
+          nextMajorVersion={'2.0'}
+          nextMinorVersion={'1.1'}
           handleClose={handleClose}
         />
       )
-      cy.findByText('Update Current Version').should('exist')
+      cy.contains('Update Current Version').should('exist')
```

Comment:
you can change all the `cy.contains` for:
`cy.findByLabelText(/Update Current Version/).should('exist')`

I think is more suitable for finding an input by its label.